docs/examples/use_cases/tensorflow/efficientdet/pipeline/dali/efficientdet_pipeline.py

```diff
@@ -0,0 +1,262 @@
+# Copyright 2021 Kacper Kluk, Paweł Anikiel, Jagoda Kamińska. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import nvidia.dali as dali
+import nvidia.dali.plugin.tf as dali_tf
+import tensorflow as tf
+import math
+
+from absl import logging
+from glob import glob
+from pipeline import anchors
+from utils import InputType
+
+from . import ops_util
+
+
+class EfficientDetPipeline:
+    def __init__(
+        self,
+        params,
+        batch_size,
+        args,
+        is_training=True,
+        num_shards=1,
+        device_id=0,
+        cpu_only=False,
+    ):
+        self._batch_size = batch_size
+        self._image_size = params["image_size"]
+        self._gridmask = params["grid_mask"]
+
+        self._input_type = args.input_type
+        if self._input_type == InputType.tfrecord:
+            if is_training:
+                file_pattern = args.train_file_pattern
+            else:
+                file_pattern = args.eval_file_pattern or args.train_file_pattern
+
+            self._tfrecord_files = glob(file_pattern)
+            self._tfrecord_idxs = [
+                filename + "_idx" for filename in self._tfrecord_files
+            ]
+        elif self._input_type == InputType.coco:
+            self._images_path = args.images_path
+            self._annotations_path = args.annotations_path
+
```

Comment:
Hmm... if there's no default input type, then I guess we should error-out?
```suggestion
            else:
              raise ValueError("Invalid input type: ", self._input_type)
```

Reply:
It can't happen. Input pipeline type is defined as an enum. Argparse will handle invalid type and return error message like: 
`train.py: error: argument --input_type: invalid choice: 'abcd' (choose from tfrecord, coco)`